sdk/storage/azdatalake/filesystem/responses.go

```diff
@@ -24,16 +26,117 @@ type SetMetadataResponse = container.SetMetadataResponse
 type SetAccessPolicyResponse = container.SetAccessPolicyResponse
 
 // GetAccessPolicyResponse contains the response from method FilesystemClient.GetAccessPolicy.
-type GetAccessPolicyResponse = container.GetAccessPolicyResponse
+type GetAccessPolicyResponse struct {
```

Comment:
rewrote it manually here since we want to remove the "blob" prefix from PublicAccess